charmcraft/commands/store/store.py

```diff
@@ -144,11 +144,9 @@ def list_registered_names(self):
                 entity_type=item['type']))
         return result
 
-    def upload(self, name, filepath):
-        """Upload the content of filepath to the indicated charm."""
+    def _upload(self, endpoint, filepath):
+        """Uploadfor all charms, bundles, resources (generic process)."""
```

Comment:
A typo here? Perhaps this should be "Upload routine for..." (or even "Common upload routine for..."?)

Reply:
I actually wrote "Generic upload..." initially, but PEP257 complained that it was not a verb in imperative...

Improved it a little, though.

Reply:
I think the PEP257 checker will complain a lot about my docstrings, I'll add it to my lint workflow.